queue_services/strr-pay/devops/gcp/clouddeploy.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: deploy.cloud.google.com/v1
+kind: DeliveryPipeline
+metadata:
+ name: strr-pay-pipeline
+description: Deployment pipeline
+serialPipeline:
+ stages:
+ - targetId: bcrbk9-dev
+   profiles: [dev]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "development"
+      deploy-project-id: "bcrbk9-dev"
+      service-name: "strr-pay-dev"
+      container-name: "strr-pay-dev"
+      cloudsql-instances: "bcrbk9-dev:northamerica-northeast1:strr-db-dev"
+      service-account: "sa-api@bcrbk9-dev.iam.gserviceaccount.com"
+ - targetId: bcrbk9-test
+   profiles: [test]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "test"
```

Comment:
there is no  deploy-env: "test", need to replace it with either "development" or "staging" (see my comment on sandbox env below)